### PR TITLE
type: fix Modal styles typing

### PR DESCRIPTION
**Pull request by afc163**
[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [x] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |  Fix Modal missing `styles.wrapper` type.         |
| 🇨🇳 Chinese | 修复 Modal `styles.wrapper` TS 定义丢失的问题。          |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/modal/__tests__/type.test.tsx [hunk 1]
```diff
@@ -4,8 +4,28 @@ import Modal from '..';
...
+  });
+
+  it('Modal.styles', () => {
+    const style: React.CSSProperties = {
+      position: 'absolute',
+    };
+    const modal = (
+      <Modal
+        styles={{
+          header: style,
```
Comment (crazyair) [inline components/modal/__tests__/type.test.tsx:19]: 这个没有样式的单测吗？还需要加个 type 的单测？是否多余了？

Comment (afc163) [reply components/modal/__tests__/type.test.tsx:19]: 目前确实还没有样式的单测。

Comment (crazyair) [reply components/modal/__tests__/type.test.tsx:19]: 应该加 dom 的单测，一次单测做2件事

Comment (afc163) [reply components/modal/__tests__/type.test.tsx:19]: 确实可以加 demo 测试 styles 的样式是否生效，不过这是两件事，可以另开 PR 来做。

**Review by crazyair — APPROVED** [review]

**Pull request merged**